gen/field.go

```diff
@@ -440,3 +444,24 @@ func (f fieldGroupGenerator) Equals(g Generator) error {
 		}
 		`, f)
 }
+
+func (f fieldGroupGenerator) PrimitiveAccessors(g Generator) error {
+	return g.DeclareFromTemplate(
+		`
+		<$v := newVar "v">
+		<$o := newVar "o">
+		<$name := .Name>
+		<range .Fields>
+			<$fname := goName .>
+			<if and (not .Required) (isPrimitiveType .Type)>
```

Comment:
Does generating the same type of accessors for non-primitive types introduce much extra complexity?

Reply:
No, but you don't gain anything by doing it. Non-primitive types are always passed around as pointers so the accessor will simply return the value of the field. `x.Foo` will be no different from `x.GetFoo()`, so users have no reason to use `GetFoo`.

Reply:
Ah, makes sense.